lib/mix/tasks/secrex.decrypt.ex

```diff
@@ -15,11 +15,16 @@ defmodule Mix.Tasks.Secrex.Decrypt do
 
     for path <- secret_files() do
       enc_path = encrypted_path(path)
-      Mix.shell().info("decrypting #{enc_path}")
-      decrypted = decrypt(enc_path, key)
-      File.write!(path, decrypted)
-    end
+      Mix.shell().info("Decrypting #{enc_path}")
+
+      case decrypt(enc_path, key) do
+        {:ok, decrypted} ->
+          File.write!(path, decrypted)
+          Mix.shell().info("Files have been decrypted")
 
-    Mix.shell().info("files decrypted")
+        {:error, reason} ->
```

Comment:
We either should make `reason` be atom or not inspect it when raising.

Reply:
converted reason to atom.